docs/workflows/common-workflow-steps/custom-script.md

```diff
@@ -227,87 +227,133 @@ sudo -E printenv
 
 ### How can I send a custom Email?
 
-Appcircle provides a ready-to-use email structure in the [**Testing Distribution**](/testing-distribution/create-or-select-a-distribution-profile#share-binary), and [**Publish**](/publish-integrations/common-publish-integrations/get-approval-via-email) modules. This structure varies across the three modules. If desired, the user can customize this structure by using the custom script below to send their own custom email.
+Appcircle provides a **ready-to-use email** structure in the [**Testing Distribution**](/testing-distribution/create-or-select-a-distribution-profile#share-binary), and [**Publish**](/publish-integrations/common-publish-integrations/get-approval-via-email) modules. This structure varies across the three modules. If desired, the user can customize this structure by using the custom script below to send their own custom email.
+
+The following script is set to use a [**Gmail SMTP Server**]. For more information, please visit [**Gmail SMTP Server**](https://support.google.com/a/answer/176600?hl=en) documentation. 
 
 ```bash
 
-os=""
-if uname -a | grep -iq "darwin"
-then
-	os="darwin"
-elif uname -a | grep -iq "linux"
-then
-	os="linux"
-fi
+# Set SMTP server 
+HOST_="smtp.gmail.com"
+PORT_="587"
+ACCOUNT_="gmail"
 
-if [ "$os" == "" ]
-then
-	echo "This script expects darwin or linux."
-	exit 1
-elif [ "$os" == "darwin" ] && which brew | grep -c "not found"
-then
-	echo "Can't find brew installation; make brew command visible or install homebrew and try again"
-	exit 1
-elif [ "$os" == "linux" ] && ! dpkg -s apt | grep -c "install ok installed"
-then
-	echo "apt is not installed; install apt and try again"
-	exit 1
+# Add account details (make sure to replace EMAIL_, USERNAME_, and PASSWORD_ with your actual details)
+EMAIL_="your-email-address@gmail.com"
+USERNAME_="your-email-address@gmail.com"
+PASSWORD_="your-email-password"
+
+# Send a test email
+EMAIL_SUBJECT="Test Email Subject"
+EMAIL_TO="recieve-email-address"
+EMAIL_FROM="sender-email-address"
+EMAIL_BODY="This is the body of the test email."
+
+# Detect operating system
+os=""
+if uname -a | grep -iq "darwin"; then
+    os="darwin"
+elif uname -a | grep -iq "linux"; then
+    os="linux"
 fi
 
-if [ "$os" == "linux" ]
-then
-	sudo apt-get install mailutils
-	sudo apt-get install msmtp
-	sudo apt-get install msmtp-mta
+# Check if OS is supported
+if [ "$os" == "" ]; then
+    echo "This script expects darwin or linux."
+    exit 1
+elif [ "$os" == "darwin" ] && ! which brew > /dev/null 2>&1; then
```

Comment:
OS is checked in 3 different places for both `darwin` and `linux`. 
This usage does not comply with the **DRY** principle.

> DRY (Don’t repeat yourself)
>
> This principle states that each small pieces of knowledge (code) may only occur exactly once in the entire system.

Could you please edit the code like the pseudocode below?

```bash
...
Create the .msmtprc file

if OS=="darwin":
   if !brew:
      "Error msg: brew not found."
      exit 1
   end
   brew install mailutils & msmtp
   write tls_fingerprint >> ~/.msmtprc
elif OS=="linux":
   if !apt:
      "Error msg: apt not found."
      exit 1
   end
    apt-get install -y mailutils msmtp msmtp-mta
    write ca-cert >> ~/.msmtprc
end
...
```